packages/guides-cli/src/Command/Run.php

```diff
@@ -261,7 +258,7 @@ private function getSettingsOverriddenWithInput(InputInterface $input): ProjectS
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $settings = $this->getSettingsOverriddenWithInput($input);
-        $inputDir = $this->getAbsolutePath($settings->getInput());
```

Comment:
I think we should keep this, line, we want to break when input is not an existing directory.

Reply:
Except that the getAbsolutePath did convert it to a real path, relative to the current working directory, I think that's an important difference.

Reply:
> Except that the getAbsolutePath did convert it to a real path
yes

> relative to the current working directory

Uuuuuh… no? :sweat_smile: I mean it's called `getAbsolutePath`, so, it produced absolute paths. 



Anyway, in the rest of the method `$inputDir` unused from these 3 lines:

```php
        if (!is_dir($inputDir)) {
            throw new RuntimeException(sprintf('Input directory "%s" was not found! ' . "\n" .
                'Run "vendor/bin/guides -h" for information on how to configure this command.', $inputDir));
        }
```

So the only difference it makes is in the error message of that exception. Is it important to have an absolute path in that message?